### ISSUE TITLE
If user tries to access part of site that they don't have access to flash a message instead of 403

### ISSUE THREAD

**Issue by levlaz** [levlaz/braindump#15]
Showing a 403 is jenky, just tell the user something is wrong. :)

**Comment by felicianotech** [comment]
This should be re-worded as "do both". The response should still be a 403, just provide a friendly, and perhaps functional message.

**Comment by levlaz** [comment]
@felicianotech Yes of course :) basically just need to add a handler for 403 that shows a template with an explanation of what went wrong.
